docs/en/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -303,7 +311,7 @@ There are several deployment methods available via the {aws} Serverless Applicat
 
 * <<aws-serverless-forwarder-deploy-console>>
 * <<aws-serverless-forwarder-deploy-cloudformation>>
-* <<aws-serverless-forwarder-deploy-terraform>>
+* <<aws-serverless-forwarder-deploy-sar-terraform>>
```

Comment:
Yes, but mainly because `build_docs` fails with two IDs that are the same:

```
INFO:build_docs:
INFO:build_docs:asciidoctor: WARNING: aws-deploy-elastic-serverless-forwarder.asciidoc: line 401: id assigned to block already in use: aws-serverless-forwarder-deploy-terraform
```